### PR TITLE
Fix backend crash and other small things

### LINKED ISSUE #21: Drinking water contains 4% alcohol in Wikidata
There must be some ingredient that is in the "drinking water" category which for some reason contains alcohol

### PR DESCRIPTION
**Pull request by Jonasdoubleyou**
resolves #21

### CODE REVIEW

File: frontend/src/basic/Card.css [hunk 1]
```diff
@@ -19,7 +19,9 @@
 }
 
 .basic-card-image {
-    height: 100%;
+    min-height: 100%;
+    min-width: 100%;
+    height: 1px;
```
Comment (Quaffel) [inline frontend/src/basic/Card.css:24]: `object-fit: cover` is easier, but should work just fine.

File: frontend/src/searchform/searchform.tsx [hunk 2]
```diff
@@ -7,26 +7,13 @@ import { useTipsySelector } from '../tipsyselector/tipsyselector';
...
     submit
 }: {
     submit(query: { weight: number, ingredients: IIngredient[], promille: number }): void
 }) {
     const [weight, setWeight] = React.useState(70);
-    const [ingredients, setIngredients] = React.useState<IIngredient[]>([]);
+    const [ingredients, setIngredients] = React.useState<IIngredient[] | null>(null);
     const [tipsySelectorEl, promille] = useTipsySelector({ rangeOptions: { min: .3, max: 2 } });
 
     // If the ingredients were loaded before the user started adding some,
```
Comment (Quaffel) [inline frontend/src/searchform/searchform.tsx:19]: As you've removed the default ingredients, this comment no longer holds true.

**Review by Quaffel — COMMENTED** [review]
What do we do about _very basic_ ingredient such as "water" and "carbonated water" then? Would be a bummer if users miss out on cocktails just because they haven't selected water explicitly!

I'd suggest to automagically add them to the fridge in the backend. This way, the user interface isn't bloated with useless items and the users are still able to select them explicitly (ensuring that they don't try to distinguish between "regular" and "basic" ingredients). We can then also adjust the default list without having to deal with fridges that were created before the update.

Feel free to merge if you don't want to address this issue now, but please create an issue referencing this message then.

**Comment by Jonasdoubleyou** [comment]
created #44

File: frontend/src/searchform/searchform.tsx (hunk 2 above)
Comment (Jonasdoubleyou) [reply frontend/src/searchform/searchform.tsx:19]: true, we will touch these parts anyways soon though, so I just leave it that way

**Pull request merged**